### PR TITLE
En 676 page settings endpoints

### PR DESCRIPTION
**Pull request by paddeo**

### CODE REVIEW

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/web/pagesettings/PageSettingsController.java [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestMethod;
+import org.springframework.web.bind.annotation.RestController;
+
+/**
+ *
+ * @author paddeo
+ */
+@RestController
+public class PageSettingsController {
```
[the remark this answers is not in the document]
Comment (joewhite101) [reply engine/src/main/java/org/entando/entando/web/pagesettings/PageSettingsController.java]: And not sure if a delete makes sense or not here?

File: engine/src/main/java/org/entando/entando/web/group/model/GroupPutRequest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+ */
+package org.entando.entando.web.group.model;
+
+import javax.validation.constraints.NotNull;
+
+/**
+ * group put payload 
+ *
+ */
+public class GroupPutRequest {
```
[the remark this answers is not in the document]
Comment (paddeo) [reply engine/src/main/java/org/entando/entando/web/group/model/GroupPutRequest.java]: class deleted

File: engine/src/main/java/org/entando/entando/web/pagesettings/PageSettingsController.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (paddeo) [reply engine/src/main/java/org/entando/entando/web/pagesettings/PageSettingsController.java]: I think we do not need neither POST nor DELETE method. at start we always have default configuration for page settings and every setting parameter is fixed: we should neither add nor remove any.

**Comment by entando-service** [comment]
All is well

File: engine/src/main/java/org/entando/entando/web/pagesettings/PageSettingsController.java (hunk 1 above)
[the remark this answers is not in the document]
Comment (joewhite101) [reply engine/src/main/java/org/entando/entando/web/pagesettings/PageSettingsController.java]: Great. Will merge

**Pull request merged**